packages/opentelemetry-exporter-collector-grpc/src/CollectorMetricExporter.ts

```diff
@@ -38,6 +39,15 @@ export class CollectorMetricExporter
   // Converts time to nanoseconds
   protected readonly _startTime = new Date().getTime() * 1000000;
 
+  constructor(config: CollectorExporterConfigNode = {}) {
```

Comment:
since you are called `super` here, shouldn't the parsing/assignment in `CollectorExporterNodeBase` works without the code you added ?

Reply:
The env var is different. Parent class assigns from `OTEL_EXPORTER_OTLP_HEADERS`, children classes then re-assign from `OTEL_EXPORTER_OTLP_METRICS_HEADERS` and `OTEL_EXPORTER_OTLP_TRACES_HEADERS`. 

Do you think there's a better pattern to accomplish this with less duplication?

Reply:
Yeah right, no i don't think there is a better way with the current structure. lgtm !